pkg/storage/test/tuples.go

```diff
@@ -1215,6 +1217,92 @@ func ReadTest(t *testing.T, datastore storage.OpenFGADatastore) {
 	})
 }
 
+func ReadPageTest(t *testing.T, datastore storage.OpenFGADatastore) {
+	ctx := context.Background()
+	storeID := ulid.Make().String()
+
+	tuplesWritten := []*openfgav1.TupleKey{
+		tuple.NewTupleKey("document:1", "reader", "user:anne"),
+		// read should skip over these
+		tuple.NewTupleKey("document:2", "a", "user:anne"),
+		tuple.NewTupleKey("document:2", "b", "user:anne"),
+		tuple.NewTupleKey("document:2", "c", "user:anne"),
+		tuple.NewTupleKey("document:2", "d", "user:anne"),
+		tuple.NewTupleKey("document:2", "e", "user:anne"),
+		tuple.NewTupleKey("document:2", "f", "user:anne"),
+		tuple.NewTupleKey("document:2", "g", "user:anne"),
+		tuple.NewTupleKey("document:2", "h", "user:anne"),
+		tuple.NewTupleKey("document:2", "j", "user:anne"),
+		// end of skip
+		tuple.NewTupleKey("document:1", "admin", "user:anne"),
+	}
+
+	err := datastore.Write(ctx, storeID, nil, tuplesWritten)
+	require.NoError(t, err)
+
+	t.Run("returns_2_results_and_no_continuation_token_when_page_size_2", func(t *testing.T) {
+		tuplesRead, contToken, err := datastore.ReadPage(
+			ctx,
+			storeID,
+			tuple.NewTupleKey("document:1", "", "user:anne"),
+			storage.PaginationOptions{
+				PageSize: 2,
+			},
+		)
+		require.NoError(t, err)
+
+		expectedTuples := []*openfgav1.Tuple{
+			{Key: tuple.NewTupleKey("document:1", "admin", "user:anne")},
+			{Key: tuple.NewTupleKey("document:1", "reader", "user:anne")},
```

Comment:
```suggestion
			tuple.NewTupleKey("document:1", "reader", "user:anne"),
```

Reply:
i assume this is a nitpick and will come back to it in a different PR since i already have approvals